src/applications/gi/sass/partials/_gi-profile-page.scss

```diff
@@ -208,9 +208,14 @@
     }
   }
 
-  .vet-tec-application-process-list {
-    li {
-      list-style: square;
+  .vet-tec-application-process {
+    ul {
+      li {
+        list-style: square;
```

Comment:
Is this necessary?  By default `<li>` items should inherit from `<ul>`, and formation should be applying `list-style: square` globally.